linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml

```diff
@@ -21,6 +21,7 @@ identifiers:
     cce@rhel7: CCE-27205-4
     cce@rhel8: CCE-80819-6
     cce@rhel9: CCE-83720-3
+    cce@sle15: CCE-85801-9
```

Comment:
Note that this CCE is reused by an earlier PR:

```
linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_etc_group/rule.yml:17:    cce@sle15: CCE-85801-9
```

Reply:
Yup! My bad, fixing it now :) thanks